src/libsync/syncjournaldb.cpp

```diff
@@ -286,6 +286,13 @@ bool SyncJournalDb::checkConnect()
         return sqlFail("Create table version", createQuery);
     }
 
+    // create the checksumtype table.
+    createQuery.prepare("CREATE TABLE IF NOT EXISTS datafingerprint("
+                               "fingerprint TEXT UNIQUE"
+                               ");");
```

Comment:
So it is really a per-oC global fingerprint?
And this will never change per storage or anything?

FYI @moscicki